Neos.ContentRepository/Classes/Service/Utility/TransientNodeCache.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
```suggestion
<?php
declare(strict_types=1);

```

Reply:
Well, it's to be introduced as of recently… https://discuss.neos.io/t/usage-of-php-strict-typing-by-default/3861

Reply:
Actually it should have a newline more…
```suggestion
<?php

declare(strict_types=1);
```